app/graph/types/team_type.rb

```diff
@@ -295,4 +295,14 @@ def tipline_messages(uid:)
   def articles(article_type:)
     object.explainers if article_type == 'explainer'
   end
+
+  field :api_key, ApiKeyType, null: true do
+    argument :dbid, GraphQL::Types::Int, required: true
+  end
+
+  def api_key(dbid:)
+    object.get_api_key(dbid)
```

Comment:
Please check if the user has permission to read the API key before returning it... something like:
```
api_key = object.get_api_key(dbid)
ability.can?(:read, api_key) ? api_key : nil
```